packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -372,198 +371,62 @@ export const testParasRococo: EndpointOption[] = [
   }
 ];
 
-// Based on history, this will expand so keep it as a singular chunk
-export function createRococo (t: TFunction): EndpointOption {
-  return {
-    dnslink: 'rococo',
-    genesisHash: ROCOCO_GENESIS,
-    info: 'rococo',
-    text: t('rpc.rococo', 'Rococo', { ns: 'apps-config' }),
-    providers: {
-      Parity: 'wss://rococo-rpc.polkadot.io',
-      'light client': 'light://substrate-connect/rococo'
-      // OnFinality: 'wss://rococo.api.onfinality.io/public-ws', // After reset, node misses host functions
-      // Pinknode: 'wss://rpc.pinknode.io/rococo/explorer' // After reset, syncs to old chain
-      // 'Ares Protocol': 'wss://rococo.aresprotocol.com' // https://github.com/polkadot-js/apps/issues/5767
+export const testParasRococoCommon: EndpointOption[] = [
```

Comment:
What's the rule used here for choosing which networks we want to support from our fork?

Reply:
I'd say upstream repo is right - so for all config files etc upstream is a golden source, ofc we then append our networks

Reply:
Got it. I reviewed it as part of the "resolve merge conflicts" commit and thought that we're removing those ourselves. If it's upstream then I get it.